src/Razor/src/Microsoft.AspNetCore.Razor.VSCode.BlazorWasmCompanionExtension/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "blazorwasm-companion",
+  "displayName": "Microsoft.AspNetCore.Razor.VSCode.BlazorWasmCompanionExtension",
+  "description": "Companion extension for debugging Blazor WebAssembly remotely",
```

Comment:
You can remove my "remotely" piece now. At first I didn't realize this was also used in local scenarios.
```suggestion
  "description": "Companion extension for debugging Blazor WebAssembly projects",
```